helm/charts/infra/templates/server/deployment.yaml

```diff
@@ -89,6 +93,11 @@ spec:
         - name: conf
           configMap:
             name: {{ include "server.fullname" . }}
+{{- if (not .Values.server.config.tls) }}
+        - name: tls-ca
```

Comment:
Is this the CA or a private key? Doesnt seem like it needs to be a secret if its just info about the CA

Reply:
The kube secret has both the CA cert and private key. The CA cert is not secret, but the private key for the CA is. I think it's easier to manage them both together in the same kube secret, like we do with the connector.  If it were just a CA cert then definitely we wouldn't need a kube secret for it.